packages/elements/src/styles/components/form.scss

```diff
@@ -1,25 +1,25 @@
-
 @import '../base/colors.scss';
 @import '../base/fonts.scss';
 
-.form {
+.form2 {
```

Comment:
Can we call this something more meaningful than form2 please?

Reply:
I accidentally changed elements styles. Proceed to revert